tests/functional/pv/pv_encryption/test_encrypted_rbd_pvc_clone.py

```diff
@@ -164,14 +165,13 @@ def test_pvc_to_pvc_clone(self, kv_version, kms_provider, pod_factory):
 
         log.info("Checking for encrypted device and running IO on all pods")
         for vol_handle, pod_obj in zip(self.vol_handles, self.pod_objs):
-            if pod_obj.exec_sh_cmd_on_pod(
-                command=f"lsblk | grep {vol_handle} | grep crypt"
-            ):
-                log.info(f"Encrypted device found in {pod_obj.name}")
-            else:
+            rbd_devices = list_encrypted_rbd_devices_onnode(pod_obj.get_node())
+            crypt_device = [device for device in rbd_devices if vol_handle in device]
+            if not crypt_device:
```

Comment:
this code repeats itself several times in the PR. Please create a function out of it and call it from all places

Reply:
Done